osquery/core/system.cpp

```diff
@@ -226,7 +226,7 @@ Status checkStalePid(const std::string& content) {
     return Status(1, "osqueryd (" + content + ") is already running");
   } else {
     VLOG(1) << "Found stale process for osqueryd (" << content
-            << ") removing pidfile";
+            << "); Removing pidfile";
```

Comment:
Just a little nitpicking, as the previous structure resulted in a bit of a run-on sentence (the action "Removing pidfile" is a consequence of the first part of the statement, and should be separated by something like a semicolon or a comma).